config/config.yaml

```diff
@@ -11,7 +11,10 @@ defaults:
     safeDnsIntAppObjectId: "c54b6bce-1cd3-4d37-bebe-aa22f4ce4fbc"
 
   # ACR Pull
-  acrPullImageDigest: sha256:51dd1a7c217b554d7925e67008caf747c8bec17b641830160e08b6836a8aa589 #v0.1.12
+  acrPull:
+    image:
+      repository: mcr.microsoft.com/aks/msi-acrpull
```

Comment:
repository is not consistently used. here it is the full qualified registry+repo reference, in other cases it is relative to the SVC ACR.